script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -956,7 +958,7 @@ The only time she was allowed to leave was when she had been assigned a new targ
 [sha:722ebbe0f00a1203607b30e5dd09650b58a42ff4]{
 -- Page 100, Offset 29784.
 -- その時に応じた知識を教えこまれて、確実に敵を処刑できるように」
-When that happened, she would be taught only as much as was necessary in order for her to execute her quarry."
+When that happened, she would be taught only as much as was necessary in order for her to execute her mark."
```

Comment:
Prefer quarry tbh

Reply:
if nobody knows what "quarry" means, I think mark is good.